src/shims/manual/mscorlib.forwards.cs

```diff
@@ -5,17 +5,12 @@
 // Add any internal types that we need to forward from mscorlib. 
 
 // These types are required for Desktop to Core serialization as they are not covered by GenAPI because they are marked as internal.
-#if netcoreapp
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.CultureAwareComparer))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.OrdinalComparer))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.OrdinalIgnoreCaseComparer))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.GenericComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.NullableComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.ObjectComparer<>))]
-[assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.Int32EnumComparer<>))]
-[assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.UInt32EnumComparer<>))]
```

Comment:
These ones only exist on CoreCLR, so they're not supposed to get serialized. The tests seem to agree.